task/sonatype/0.1/nexus-lifecycle-scan/nexus-lifecycle-scan-task.yml

```diff
@@ -0,0 +1,127 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: nexus-lifecycle-scan-task
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
```

Comment:
@bigspotteddog the directory structure is not as defined in TEP-0003. The directory structure being followed here is 
```
task
   /sonatype
      /0.1
           /nexus-lifecycle-scan
                /nexus-lifecycle-scan-task.yaml
```
ideally, it should have been like
```
task
   /nexus-lifecycle-scan-task
      /0.1
           /nexus-lifecycle-scan-task.yaml
```

Reply:
@vinamra28 , here is how we approached it.  the directory "sonatype" is a holder for all of the tasks that will be built for sonatype tooling.  The subdirectory, such as "nexus-lifecycle-scan" is the collection of resources specific to a task for invoking a nexus lifecycle scan; "Nexus Lifecycle" is one of the products.

In the future, we will have other subdirectories for other tasks specific to the broader portfolio of products;

Reply:
is this an acceptable approach?

Reply:
Actually...cancel that comment.  I'm going to update to reflect your feedback.  Thank You!

Reply:
@vinamra28 , I just made the changes to the directory structure and associated information.  Would you mind a review and approval for this?